index.html

```diff
@@ -6,35 +6,35 @@
         content="width=device-width, user-scalable=no, minimum-scale=1, maximum-scale=1,initial-scale=1" />
     <meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
     <script>window.version = 4;</script>
```

Comment:
Better to keep the script and css paths relative. That's what enables you to test your stuff in the subdirectory. When you've got an index.html down in the /2/ directory, it should pull in the scripts and css under /2/, not the ones at the top.